packages/map/src/map/index.jsx

```diff
@@ -66,12 +76,14 @@ Map.defaultProps = {
   lockedMobile: true,
   noAttribution: false,
   fitPadding: 20,
+  translations: {},
```

Comment:
no need for default props if prop is required

Reply:
don't want to pass translation object in every story

Reply:
i would prefer to do something like `const lang = props.translations || defaultTranslations` in render